### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Certificates_Keyboard: Focus is not visible on the "More (...)", "Refresh" and "Details" button

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#169]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0 , Google Chrome Version 68.0.3440.106 
Screen Reader: Narrator, NVDA
MAS ID Violated: MAS2.4.7

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Certificates and press enter
7) Verify that bold frame(dotted or plain) is there to distinguish between the focused elements on the page.

**Actual Result:**
Focus is not visible on the "More (…)", "Refresh" and "Details" button

**Expected Result:**
Focus should be visible clearly on all the controls on page.

**User Impact:** 
Users will face difficulty in navigating through the contents and controls if the focus is not visible properly

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachments:**
![bug_169_focus visible_issue](https://user-images.githubusercontent.com/42638915/44782118-c6e95700-aba3-11e8-9916-1021028c4e32.jpg)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.4.7; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
